zbrowse.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
`index.js` isn't in the same directory as this script, could you prepend it with `js` so it's `node js/index.js $@`? Otherwise, this looks great!